src/lib/loadEventsNewsMetadataMap.ts

```diff
@@ -1,10 +1,13 @@
 import gql from "graphql-tag";
 import { print as printQuery } from "graphql";
+import constructEventSlug from "./util/constructEventSlug";
 import type { ContentfulClient } from "$lib/services/server/contentful";
-import type { MetadataMapItem } from "./types";
+import type { PageMetadataMapItem } from "$lib/loadPageMetadataMap";
+
+import type { NewsAndEventsCollectionQuery } from "./$queries.generated";
 
 const queryNewsAndEvents = gql`
-  query NewsCollection {
+  query NewsAndEventsCollection {
```

Comment:
We were missing the new meta fields below in the query. We should also be using `eventSummary` over `eventDescription`.